doc/src/manual/performance-tips.md

```diff
@@ -805,36 +805,36 @@ do this in at least four ways (in addition to the recommended call to the built-
 
 ```julia
 function copy_cols(x::Vector{T}) where T
-    n = size(x, 1)
-    out = Array{T}(n, n)
-    for i = 1:n
+    n = indices(x, 1)
```

Comment:
Maybe change the name of the variable to e.g. `inds`.

Reply:
Yes, good point. I made the change.